core/controllers/classroom_test.py

```diff
@@ -381,6 +605,74 @@ def test_update_classroom_data(self) -> None:
 
         self.logout()
 
+    def test_update_classroom_thumbnail_data_only(self) -> None:
```

Comment:
It's not clear what this is testing and how it relates to the test below. Can you use a better name to clarify?

Also I see no assertions in the test, so I'm not sure what behaviour we're trying to verify here.